interface/modules/custom_modules/oe-module-faxsms/contact.php

```diff
@@ -0,0 +1,220 @@
+                    let wait = '<div class="text-center wait"><i class="fa fa-cog fa-spin fa-2x"></i></div>';
+                    let url = 'sendFax?type=fax';
+                    if (isSms) {
+                        url = 'sendSMS?type=sms';
+                    }
+                    $('#contact-form').find('.messages').html(wait);
+                    // POST values in the background the script URL
+                    $.ajax({
+                        type: "POST",
+                        url: url,
+                        data: $(this).serialize(),
+                        success: function (data) {
+                            let err = (data.search(/Exception/) !== -1 ? 1 : 0);
+                            if (!err) {
+                                err = (data.search(/Error:/) !== -1) ? 1 : 0;
+                            }
+                            // Type of the message: success or danger. Apply it to the alert.
+                            let messageAlert = 'alert-' + (err !== 0 ? 'danger' : 'success');
+                            let messageText = data;
+
+                            // let's compose alert box HTML
+                            let alertBox = '<div class="alert ' + messageAlert + ' alert-dismissable">' +
+                                '<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>' + messageText + '</div>';
+
+                            // If we have messageAlert and messageText
+                            if (messageAlert && messageText) {
+                                // inject the alert to messages div in our form
+                                $('#contact-form').find('.messages').html(alertBox);
+                                setTimeout(function () {
+                                    if (!err) {
+                                        // close dialog as we have success.
+                                        dlgclose();
+                                    }
+                                    // if error let user close dialog for time to read error message.
+                                }, 3000);
+                            }
+                        }
+                    });
+                    return false;
+                }
+            })
+        });
+
+        function sel_patient() {
+            const url = top.webroot_url + '/interface/main/calendar/find_patient_popup.php?pflag=0&pkretrieval=1'
+            dlgopen(url, '_blank', 'modal-md', 500);
+        }
+
+        function setpatient(pid, lname, fname, dob) {
+            $("#form_patient").val(fname + " " + lname);
+            $("#form_patient_id").val(fname + "" + pid);
+            $("#form_pid").val(pid);
+        }
+
+        function contactCallBack(contact) {
+            let actionUrl = 'getUser';
+            return $.post(actionUrl, {
+                'uid': contact,
+                'type': <?php echo js_url($serviceType); ?>
```

Comment:
jquery/fetch will url encode parameters when done like this, so should just need `js_escape`